server/models/StudiesModel/index.js

```diff
@@ -10,6 +10,32 @@ const StudiesModel = {
       .filter((study) => !STUDIES_TO_OMIT.includes(study))
       .sort((prevStudy, nextStudy) => (prevStudy < nextStudy ? -1 : 1))
   },
+  index: async (db, query) =>
+    await db.collection(collections.metadata).aggregate(query).toArray(),
 }
 
+export const studiesTableQuery = (studiesList, sortDirective) => {
```

Comment:
why does this method exist outside of the model object, while all the others are attributes of that object?

Reply:
That's been the pattern that was set from a pr comment long ago. We do this for charts 
https://github.com/AMP-SCZ/dpdash/blob/32550c70c74884d7ea2c1a654c04212b672f31b1/server/models/ChartsModel/index.js#L44
participants
https://github.com/AMP-SCZ/dpdash/blob/32550c70c74884d7ea2c1a654c04212b672f31b1/server/models/ParticipantsModel/index.js#L68